publicationiq.json

```diff
@@ -337,6 +337,82 @@
         }
       ]
     },
+    "/pf/pfaccount/{profile}/package/{packageid}": {
+      "get": {
+        "tags": [
+          "Package Resources"
+        ],
+        "summary": "Get Package by package ID",
+        "description": "This resource allows you to retrieve a package by package ID.  Access to PublicationIQ requires a Publication Finder customer profile.  A profile password is optional.  If a password is supplied, it must be a valid password assigned to the Publication Finder customer profile.  If the password is omitted, the request will only be honored if the Publication Finder customer profile allows guest access in the environment where you are making the request.  For example, if you would like to make a request to PublicationIQ in the sandbox environment without a password, guest access must be enabled for the Publication Finder customer profile in the sandbox environment.  The same is true in the production environment.  This documentation uses the sandbox environment.  In order to gain access to the PublicationIQ sandbox environment, please contact EBSCO customer support.",
+        "operationId": "getPackageUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "packageid",
+            "in": "path",
+            "description": "Package ID",

```

Comment:
Can you add an example package id similar to kbid? https://github.com/pdestefano/developer_portal_openapi/blob/65f1df742f19b6650248f02118d2c54e62ca5d96/publicationiq.json#L785

That would let user know better that it's numeric.